Zotero/Scenes/Detail/PDF/Views/PDFDocumentViewController.swift

```diff
@@ -998,25 +1039,59 @@ extension PDFDocumentViewController: FreeTextInputDelegate {
 }
 
 final class SelectionView: UIView {
-    static let inset: CGFloat = 4.5 // 2.5 for border, 2 for padding
+    private static let inset: CGFloat = 4.5 // 2.5 for border, 2 for padding
 
-    init() {
-        super.init(frame: CGRect())
-        self.commonSetup()
+    override init(frame: CGRect) {
+        super.init(frame: frame.insetBy(dx: -Self.inset, dy: -Self.inset))
+        commonSetup()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonSetup()
+    }
+
+    private func commonSetup() {
+        backgroundColor = .clear
+        autoresizingMask = [.flexibleTopMargin, .flexibleLeftMargin, .flexibleBottomMargin, .flexibleRightMargin, .flexibleWidth, .flexibleHeight]
+        layer.borderColor = Asset.Colors.annotationHighlightSelection.color.cgColor
+        layer.borderWidth = 2.5
+        layer.cornerRadius = 2.5
+        layer.masksToBounds = true
+    }
+}
+
+final class AnnotationPreviewView: UIView {
```

Comment:
`AnnotationPreviewView` shares a lot of code with `SelectionView` and can probably be made a subclass of that one in the future.

Reply:
6c880b9f408df98043278cb720159169778e2696